synapse/module_api/__init__.py

```diff
@@ -880,6 +880,16 @@ async def update_room_membership(
             content=content,
         )
 
+        # Update the m direct list
+        is_direct = content.get("is_direct", None)
+        logger.debug("InviteAutoAccepter: is_direct is %s", is_direct)
+        if is_direct:
```

Comment:
I notices a log where the sender and target are the same, is this somehow because there is also an event for the user who created the room, should I exclude such cases?
  `if target not is sender and is_direct`

Reply:
Ah yes sorry I probably forgot to add the content part being passed to this pull request. so I guess I do need that check.